### PR TITLE
Week2_Sharath Balaji T

### PR DESCRIPTION
**Pull request by sharath-2003**

### CODE REVIEW

File: week_2/pet.rb [hunk 1]
```diff
@@ -90,7 +95,11 @@ def food_cost(days)
   # snake = Pet.new(name: 'python', animal_type_id: 4, food_consumed_per_day: 0.3)
   # Pet.cost_to_feed([cat, dog, fish, snake], 6) will return 6180.0
   def self.cost_to_feed(pets, days)
-    raise NotImplementedError # TODO
+    ttpd=0
+    for pet in pets
+        ttpd+=pet.food_cost(days)
+     end
+    return ttpd
```
Comment (hrushikeshj) [inline week_2/pet.rb:102]: [reduce](https://medium.com/@james.a.hughes/using-the-reduce-method-in-ruby-907f3c18ae1f) method cab be used here.

File: week_2/pet.rb [hunk 2]
```diff
@@ -110,6 +119,13 @@ def self.cost_to_feed(pets, days)
...
-    raise NotImplementedError # TODO
+    grp = {}
+    pets.group_by { |pet| pet.animal_type_id }.each do |animal_type_id, pets|
+    names = []
+    pets.each { |pet| names << pet.name }
+    grp[animal_type_id] = names
   end
+   return grp
+   end
+
```
Comment (hrushikeshj) [inline week_2/pet.rb:130]: It is good practice to format code.

**Review by hrushikeshj — APPROVED** [review]
Good job

File: week_2/pet.rb (hunk 1 above)
Comment (sharath-2003) [reply week_2/pet.rb]: changes are made, please verify it

**Pull request closed without merging**